package.json

```diff
@@ -85,11 +87,13 @@
     "node": ">=16"
   },
   "resolutions": {
-    "@types/react": "^17",
-    "@types/react-native": "~0.64.29",
-    "react": "17.0.1",
-    "react-dom": "17.0.1",
-    "react-native": "0.64.2"
+    "@mendix/pluggable-widgets-tools": "/Users/Diego.Antonelli/Desktop/projects/widgets-tools/packages/pluggable-widgets-tools/mendix-pluggable-widgets-tools-9.23.2.tgz",
```

Comment:
this is blocking me from building the widgets from my end, is there another way to depend on a GH branch or store the file in an accessible location instead of the local path?